cni/network/network_windows_test.go

```diff
@@ -301,3 +301,59 @@ func TestDSRPolciy(t *testing.T) {
 		})
 	}
 }
+
+func TestGetNetworkNameFromCNS(t *testing.T) {
+	plugin, _ := cni.NewPlugin("name", "0.3.0")
+	tests := []struct {
+		name  			string
+		plugin      	*NetPlugin
+		netNs 			string
+		nwCfg 			*cni.NetworkConfig
+		ipamAddResult   *IPAMAddResult
+		want 			string
```

Comment:
I don't know why the linter is not yelling at you for this, but it needs to be gofmt

Reply:
I used tab here and has configured my tab to be 3 spaces! for some reason it didn't follow through. Will use space instead.